FrostyEditor/Windows/MainWindow.xaml.cs

```diff
@@ -420,15 +426,15 @@ private void launchButton_Click(object sender, RoutedEventArgs e)
                         task.Update("Exporting Mod");
                         ExportMod(editorSettings, $"Mods/{ProfilesLibrary.ProfileName}/{editorModName}", true);
                         modPaths.Add(editorModName);
+                        App.Logger.Log("Editor temporary Mod saved to {0}", $"Mods/{ProfilesLibrary.ProfileName}/{editorModName}");
 
                         // allow cancelling in case of a big mod (will cancel after processing the mod)
                         // @todo: add cancellation to different stages of mod exportation, to allow cancelling
                         //        at any stage of a large mod
 
                         cancelToken.Token.ThrowIfCancellationRequested();
-
                         task.Update("");
-                        executor.Run(App.FileSystem, cancelToken.Token, task.TaskLogger, $"Mods/{ProfilesLibrary.ProfileName}/", App.SelectedPack, additionalArgs.Trim(), modPaths.ToArray());
+                        executor.Run(App.FileSystem, cancelToken.Token, task.TaskLogger, $"Mods/{ProfilesLibrary.ProfileName}/", App.SelectedPack, additionalArgs.Trim(), true, modPaths.ToArray());
```

Comment:
i think it would be better to delete the mods.json file in the editor than adding this boolean

Reply:
Sorry, idk how